extension/src/cli/index.ts

```diff
@@ -101,6 +96,60 @@ export class Cli extends Disposable implements ICli {
     }
   }
 
+  protected createBackgroundProcess(options: ProcessOptions) {
+    const { baseEvent, stopWatch } = this.getProcessDetails(options)
+    try {
+      const backgroundProcess = this.createProcess(baseEvent, {
+        detached: true,
+        ...options
+      })
+
+      return new Promise<string>(resolve => {
+        const readable = backgroundProcess.all
+        readable?.on('data', chunk => {
+          resolve(
+            chunk
+              .toString()
+              .split(/(\r?\n)/g)
+              .join('\r')
+          )
+          if (backgroundProcess.connected) {
```

Comment:
See [this answer](https://stackoverflow.com/questions/59646654/detach-a-spawn-child-process-after-the-start).